js/form4/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>User List</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body onload="getUsers()">
+    <h1>User List</h1>
+    <div id="table"></div>
```

Comment:
This div is container of table so id of this div can be **tableContainer** instead of **table**